svg.js.d.ts

```diff
@@ -1,3 +1,8 @@
+export = svgjs;
+export as namespace svgjs;
+
+declare var svgjs: svgjs.Library;
+

```

Comment:
The reason for that is that `svg.js` does not contain a `default` export. As a result we need to import the entire module content as an object, which is the corresponding es6 syntax. [1]

> could you explain what this code dose?

> … anywhere the module keyword was used when declaring an internal module, the namespace keyword can and should be used instead. [2]

It makes the `svgjs` namespace available globally if you use the `/// <reference` directive. [3]

The rest of the code is doing the same as before, if you replace the word namespace with module.

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Description
[2] https://www.typescriptlang.org/docs/handbook/namespaces.html
[3] https://github.com/Microsoft/TypeScript/issues/7125